src/HeroContent/heroContent.js

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import defaultStyles from './heroContent.css'
+import { Heading, Text, Image } from '@adobe/react-spectrum'
+
+/**
+ * Renders a large,  banner-like component with a title, description, and image
+ * @param {HeroContentProps} props 
+ */
+const HeroContent = (props) => {
```

Comment:
We've stopped using default props since we are using function components. So this line can become:

```javascript
const HeroContent = ({ title, children, imagePath, bgColor = '#381f81', textColor = '#e7e7e7' }) => {
```

Reply:
I thought `defaultProps` worked with functional components?

Reply:
Oh they do but we just stopped using them style wise.